raftify/src/raft_node/mod.rs

```diff
@@ -1147,7 +1147,10 @@ impl<
                 tx_msg
                     .send(LocalResponseMsg::GetRawNode {
                         raw_node: Arc::new(Mutex::new(unsafe {
-                            std::mem::transmute(&self.raw_node)
+                            std::mem::transmute::<
+                                &jopemachine_raft::RawNode<HeedStorage>,
+                                &jopemachine_raft::RawNode<HeedStorage>,
```

Comment:
Could you explain why this change is necessary?

Reply:
To clarify the namespace, but i think it's unnecessary